rfcs/search_relevancy_sandbox/20230331-project_proposal_search_relevancy_sandbox.md

```diff
@@ -0,0 +1,155 @@
+# Project Proposal: Search Relevancy Sandbox
+
+## Reviewers
+
+<!-- Choose two people at your discretion who make sense to review this based on their existing expertise. Check in to make sure folks aren't currently reviewing more than one other proposal or RFC. -->
+
+- [ ] dhruvkb
+- [ ] sarayourfriend
+
+## Project summary
+
+<!-- A brief one or two sentence summary of the project's features -->
+
+Develop mechanisms and workflows for sustainably creating & updating our staging
+API database & Elasticsearch cluster, in order to enable testing changes to
+Elasticsearch indices. The staging setup will reflect production in two ways:
+proportionally by provider and with production-level volumes.
```

Comment:
> The staging setup will reflect production in two ways:
proportionally by provider and with production-level volumes.

This might be a nit but this sentence reads as confusing to me. If staging is provider and volume-proportionate to production, it would be identical to production! Maybe some rewording to suggest that it can be in _either_ of these states for testing purposes would be useful? In general, the concept that the staging ES cluster can be adapted to have different results at different times is probably worth highlighting in this section.

Reply:
Okay! I'll try to reword this a bit 🙂